### DISCUSSION TITLE
Adding Hover functions

### DISCUSSION THREAD

**Discussion by RougherO** [noib3/cokeline.nvim#185] [General]
I tried to replicate the hover functionality to enable 'x' mark only when hovered over the buffers however it seems the hover function is not working correctly even though I used is_hovered property of the buffers they seem to be always false. I would like to know how to implement hovering.

**Comment by willothy** [comment]
Have you enabled `vim.o.mousemoveevent`? (`:h 'mousemoveevent'`)

**Reply by RougherO** [reply]
Yes this is what I was missing, now everything is working as expected....

Thank you so much awesome work :heart:

**Reply by willothy** [reply]
Awesome, glad that worked :)
